docs/manage/reference/viam-agent/_index.md

```diff
@@ -274,9 +278,13 @@ You can also start `viam-agent` in fast start mode by setting `VIAM_AGENT_FAST_S
 <!-- prettier-ignore -->
 | Name       | Type | Required? | Description |
 | ---------- | ---- | --------- | ----------- |
+| `bluetooth_adapter_name` | string | Optional | For machines with multiple Bluetooth adapters, specify which adapter to use for Bluetooth provisioning. If not specified, the first available Bluetooth adapter will be used. Example: `"hci0"`. Default: `""` (auto-detect). |
 | `device_reboot_after_offline_minutes` | integer | Optional | If set, `viam-agent` will reboot the device after it has been offline for the specified duration. Default: `0` (disabled). |
+| `disable_bt_provisioning` | boolean | Optional | When set to true, disables Bluetooth provisioning. The machine will not advertise Bluetooth services for provisioning. Both WiFi hotspot and Bluetooth provisioning can be enabled simultaneously. Default: `false`. |
 | `disable_captive_portal_redirect` | boolean | Optional | By default, ALL DNS lookups using the provisioning hotspot will redirect to the device. This causes most phones/mobile devices to automatically redirect the user to the captive portal as a "sign in" screen. When disabled, only domains ending in .setup (ex: viam.setup) will be redirected. This generally avoids displaying the portal to users and is mainly used in conjunction with a mobile provisioning application workflow. Default: `false`. |
-| `fragment_id` | string | Optional | The `fragment_id` of the fragment to configure machines with. Required when using the Viam mobile app for provisioning. The Viam mobile app uses the fragment to configure the machine. |
+| `disable_wifi_provisioning` | boolean | Optional | When set to true, disables WiFi hotspot provisioning. The machine will not create a WiFi hotspot for provisioning. Both WiFi hotspot and Bluetooth provisioning can be enabled simultaneously. Default: `false`. |
+| `bluetooth_trust_all` | boolean | Optional | When true, the machine accepts all Bluetooth pairing requests for sharing the internet from a mobile device (tethering) without requiring devices to be added during provisioning first. Default: `false`. |
+| `fragment_id` | string | Optional | The `fragment_id` of the fragment to configure machines with. Required when using Bluetooth or the Viam mobile app for provisioning. The Viam mobile app uses the fragment to configure the machine. |
```

Comment:
Should check with the mobile team on this. AFAIK, this CAN be used by a mobile app, but current implementations aren't using it.

It is definitely in no way bluetooth specific though, so not sure why or where that idea came from.
```suggestion
| `fragment_id` | string | Optional | The `fragment_id` of the fragment to configure machines with. Can be used by a mobile app (during provisioning) to automatically configure a newly created machine. |
```

Reply:
I think you're right